config.js

```diff
@@ -287,6 +288,7 @@ config.STATISTICS_COLLECTOR_EXPIRATION = 31 * 24 * 60 * 60 * 1000; // 1 month
 config.SERVICES_TYPES = Object.freeze(['AWS', 'AZURE', 'S3_COMPATIBLE', 'GOOGLE', 'FLASHBLADE']);
 config.USAGE_AGGREGATOR_INTERVAL = 24 * 60 * 60 * 1000; // 24 hours
 
+config.USAGE_UPDATE_INTERVAL = 1 * 60 * 1000; // 1 minute
```

Comment:
I suggest a more specific name:
```suggestion
config.BLOCK_STORE_USAGE_INTERVAL = 1 * 60 * 1000; // 1 minute
```